packages/team-management/src/components/TeamsAddEdit/index.tsx

```diff
@@ -107,7 +107,9 @@ export const TeamsAddEdit: React.FC = () => {
         <title>{params.id ? 'Edit' : 'Create'} Team</title>
       </Helmet>
 
-      <h5 className="mb-3">{params.id ? 'Edit' : 'Create'} Team</h5>
+      <h5 className="mb-3 header-title">
+        {params.id ? `Edit Team | ${initialValue?.name}` : 'Create Team'}
```

Comment:
If `initialValue` is `null`, we will have the heading display as **Edit Team |**. I think we should update accordingly to  make sure **|** does not show if `initialValue` is `null`